hyperbook/book/spiele/penguins/xml.md

```diff
@@ -131,8 +131,18 @@ Der Inhalt der `field` Objekte steht für die Anzahl Fische auf dem Feld, oder f
 
 `y`: `<list>` index in `<board>`
 
+##### Koordinatensysteme
+
 Beachtet, dass der XML-Status [odd-r Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-offset) nutzt, und dass die Koordinaten in Zügen als [doubled Koordinaten](https://www.redblobgames.com/grids/hexagons/#neighbors-doubled) interpretiert werden.
 
+Die Formeln für die Transformation von dem einen System in das andere sind folgendermaßen:
+
+odd-r zu doubled: ``(x, y) => (x * 2 + y % 2, y)``
+
+doubled zu odd-r: ``(x, y) => (x / 2 - y % 2, y)``
+
+Beachtet weiter, dass im doubled Koordinatensystem unabhängig der Größe (0, 1), (0, 3), (0, 5), etc. immer out of bounds sind, da sie in odd-r zu (-1, y) übersetzt werden.
```

Comment:
> Beachtet weiter, dass im doubled Koordinatensystem unabhängig der Größe, die Koordinaten (0, 1), (0, 3), (0, 5) et cetera, immer unzulässig sind, da sie in odd-r zu (-1, y) übersetzt werden.

Maybe you should wrap this information in a [alert](https://hyperbook.openpatch.org/elements/alert):
```
:::alert{warn}
Warn
:::
```

Reply:
Done